ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/TransactionService.scala

```diff
@@ -668,16 +731,17 @@ class TransactionService(session: LedgerSession) extends LedgerTestSuite(session
       "TXRejectMultiActorExcessiveAuth",
       "Reject exercising a multi-actor choice with too many authorizers") { context =>
       for {
-        ledger <- context.participant()
-        Vector(operator, receiver, giver) <- ledger.allocateParties(3)
-        agreementFactory <- ledger.create(giver, AgreementFactory(receiver, giver))
-        agreement <- ledger
+        Vector(alpha, beta) <- context.participants(2)
+        Vector(operator, receiver) <- alpha.allocateParties(2)
+        giver <- beta.allocateParty()
+        agreementFactory <- beta.create(giver, AgreementFactory(receiver, giver))
+        agreement <- alpha
           .exerciseAndGetContract[Agreement](
             receiver,
             agreementFactory.exerciseAgreementFactoryAccept)
         triProposalTemplate = TriProposal(operator, giver, giver)
-        triProposal <- ledger.create(operator, triProposalTemplate)
-        failure <- ledger
+        triProposal <- alpha.create(operator, triProposalTemplate)
+        failure <- beta
```

Comment:
Potential flakiness between `alpha` and `beta`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2900/commits/1f22a2a4fc6c6a84fe27096298c9addbafcaa327